podcasts/NowPlayingPlayerItemViewController+Shelf.swift

```diff
@@ -224,10 +223,24 @@ extension NowPlayingPlayerItemViewController: NowPlayingActionsDelegate {
     @objc func overflowTapped() {
         let shelfController = ShelfActionsViewController()
         shelfController.playerActionsDelegate = self
-        let bottomSheet = MDCBottomSheetController(contentViewController: shelfController)
-        roundCorners(bottomSheet: bottomSheet)
 
-        present(bottomSheet, animated: true, completion: nil)
+        if let sheetController = shelfController.sheetPresentationController {
+            if #available(iOS 16.0, *) {
+                // We create a custom detent height of a bit more than half the hosting VC to try and
+                // ensure that all of the shelf content is visible in the sheet. We offer a large detent
+                // as a fallback so that the user can pull the sheet up if any content is ever cut off.
+                let hostingControllerHeight = view.bounds.height
+                let sheetDetentHeight = hostingControllerHeight * 0.675
```

Comment:
> it will be better to have it scrollable to future proof new additions to this screen

Yeah, we should definitely have this so we don't have to worry about the count changing.

Would it make sense to use a larger size for smaller devices so it's less likely to be cut off? We could always check the trait collection in the custom detent provider.